### DISCUSSION TITLE
Does D3 Support JSON (Json Arrays + Json Objects etc.) as Data Source?

### DISCUSSION THREAD

**Discussion by MichaelCropper** [d3/d3#3652] [Q&A]
Can't find a working example of this when searching around. 

Seems everything required either a CSV data source, or, a multi-dimensional JavaScript Array which is just a pain to create from multiple programming languages. 

Ultimately all that is needed, from what I understand, is a basic Array of Key:Value pairs

Would be great to find an example of this so I can understand possibilities.

**Comment by Fil** [comment]
https://github.com/d3/d3-fetch#json
